library/classes/Prescription.class.php

```diff
@@ -376,9 +376,7 @@ function get_quantity()
 
     function set_route($route)
     {
-        if (is_numeric($route)) {
-            $this->route = $route;
-        }
+        $this->route = $route;
```

Comment:
This seems purposeful where someone though it important enough to do the numeric check